### ISSUE TITLE
Training Setting

### ISSUE THREAD

**Issue by HalvesChen** [TencentARC/SEED-Voken#45]
Hi,
I notice that your method requires much more epoch(330) than other methods. Have you compared the reconstruction performance of the common setting (epoch=50)?

**Comment by ShiFengyuan1999** [comment]
Hi @HalvesChen, we follow MAGVIT-v2 and adopt a long training. VQ methods with small codebook size (like 1024 and 4096) and low code dimension (e.g., 8 or 16) may saturate quickly in reconstruction performance, thus requiring fewer training epochs. IBQ gains further improvements due to the longer training, benefitting from large model compacity, i.e., large-size codebook, high-dimension codes, large model size, and high codebook utilization.
